integrations/spark/README.md

```diff
@@ -22,8 +22,143 @@ implementation 'io.github.marquezproject:marquez-spark:0.15.0'
 
 ## Getting started
 
-### Dataproc
+### Quickstart
+The fastest way to get started testing Spark and Marquez is to use the docker-compose files included
+in the project. From the spark integration directory ($MARQUEZ_ROOT/integrations/spark) execute
```

Comment:
I guess docker-compose should be run from root directory, cause you have relative path in volumes.
and it should be run like this
`docker-compose -f ./docker-compose.yml -f ./integrations/spark/docker-compose.yml up`

Reply:
> I guess, docker-compose should be run from root directory, cause you have relative path in volumes:
> 
>     * ./docker/notebooks:/home/jovyan/notebooks
> 
>     * ./integrations/spark/build:/home/jovyan/marquezlib
> 
> 
> and it should be run like this
> `docker-compose -f ./docker-compose.yml -f ./integrations/spark/docker-compose.yml up`

It *should* work from the integration directory as described in the doc. The `.env` file in the commit concatenates both the root Marquez docker-compose.yml file as well as the one in the spark integration directory. It should be enough to simply execute `docker-compose up` from the spark directory and both files should be read. Docker will follow the relative paths and mount the directories in the appropriate location. Please try it out on your machine and let me know if it doesn't work as described. It does seem to work on Mac with Docker v20.10.5.

Reply:
Few thing what I've noticed following instruction
1. If I run 
```
from pyspark.sql import SparkSession
spark = (SparkSession.builder.master('local')
         .appName('sample_spark')
         .config('spark.jars.packages', 'io.github.marquezproject:marquez-spark:0.15.+,org.postgresql:postgresql:42.2.+')
         .config('spark.extraListeners', 'marquez.spark.agent.SparkListener')
         .config('openlineage.url', 'http://marquez:5000/api/v1/namespaces/spark_integration/')
         .getOrCreate())
```
I'm getting 

`Caused by: java.lang.IllegalArgumentException: requirement failed: marquez.spark.agent.SparkListener is not a subclass of org.apache.spark.scheduler.SparkListenerInterface. `

Do you have such issue or you install local version to maven repo?

2. Related to local jar and order of running comands
As _/integrations/spark/build_ folder is generated, if I run `docker-compose up` before `gradle shadowJar`, docker will create read only empty _build_ folder.
Basically before running docker-compose, build folder should exist, so we should run  `gradle shadowJar `before `docker-compose up`
Should we specify it?

P.S. Relative path is working like a charm, didn't notice .env configuration

Reply:
Ah yeah, I assumed you'd run `gradle shadowJar` before running `docker-compose up`, but I'll specify that.

Re: the `IllegalArgumentException`, this step won't actually work until https://github.com/MarquezProject/marquez/pull/1389 is merged and 0.15.1 is released. I would like to have written this as `[0.15.1,)`, as supported by [Ivy](https://ant.apache.org/ivy/history/2.1.0/settings/version-matchers.html), but it doesn't seem to work in Spark :(